res/controllers/Denon-MC7000-scripts.js

```diff
@@ -889,6 +1036,34 @@ MC7000.censor = function(channel, control, value, status, group) {
         }
     }
 };
+// Param Button for Pitch Play to decrease pitch, or decrease star rating otherwise
+MC7000.StarsDown = function(channel, control, value, status, group) {
+    var deckNumber = script.deckFromGroup(group);
+    var deckIndex = deckNumber - 1;
+    if (value >= 0x00) {
+        if (MC7000.PADModePitch[deckIndex]) {
+            for (var i = 0; i < 8; i++) {
```

Comment:
```suggestion
            for (var padIdx = 0; padIdx < 8; padIdx++) {
```

Reply:
done

Reply:
Could you please look for other places in the code, where the same renaming makes sense.

Reply:
done